src/programs/city/mine.js

```diff
@@ -100,7 +101,34 @@ class CityMine extends kernel.process {
       return
     }
 
-    const storage = this.room.storage
+    let storage = false
```

Comment:
Should the `storage` variable be used on line 141, changing it from `this.data.ssp[source.id] = this.room.findPath(this.room.storage, source, {` to `this.data.ssp[source.id] = this.room.findPath(storage, source, {`?

Reply:
We want the distance to be relative to the storage, so if there is no storage we should wait to calculate it. I cleaned up this code so it'll work with a default value (80) should there be no storage available yet.